x-pack/legacy/plugins/snapshot_restore/public/app/app.tsx

```diff
@@ -7,6 +7,7 @@
 import React, { useContext } from 'react';
 import { Redirect, Route, Switch } from 'react-router-dom';
 import { EuiPageContent } from '@elastic/eui';
+import chrome from 'ui/chrome';
```

Comment:
Suggestion: instead of importing this here, we can use the one imported in `x-pack/legacy/plugins/snapshot_restore/public/shim.ts` and add it to context in `x-pack/legacy/plugins/snapshot_restore/public/app/index.tsx` in the `getAppProviders` function. Then consume it in `app.tsx` from context. What do you think?

Reply:
great point - fixed!